pkg/kor/kor.go

```diff
@@ -56,19 +61,26 @@ func GetKubeClient(kubeconfig string) *kubernetes.Clientset {
 	return clientset
 }
 
-func SetNamespaceList(namespace string, kubeClient *kubernetes.Clientset) []string {
-	var namespaces []string
-	if namespace != "" {
-		namespaces = append(namespaces, namespace)
-	} else {
-		namespaceList, err := kubeClient.CoreV1().Namespaces().List(context.TODO(), metav1.ListOptions{})
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Failed to retrieve namespaces: %v\n", err)
-			os.Exit(1)
-		}
-		for _, ns := range namespaceList.Items {
-			namespaces = append(namespaces, ns.Name)
+func SetNamespaceList(s NamespaceSelector, kubeClient *kubernetes.Clientset) []string {
```

Comment:
Can you rename the variable to something more meaningful?
IncludeExcludeLists or something similar